.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+[Description of PR]
+
+TODO:
+* [ ] Unit tests and/or doctests in docstrings
+* [ ] ``tox -e py36`` passes locally
+* [ ] ``tox -e py27`` passes locally
+* [ ] Docstrings and API docs for any new/modified user-facing classes and functions
+* [ ] New/modified features documented in docs/tutorial.rst
+* [ ] Changes documented in docs/release.rst
+* [ ] ``tox -e docs`` passes locally
```

Comment:
Similar story as above. Here some comment about the docs building ok.

Reply:
done